test/dynamicFormControlGroup.test.ts

```diff
@@ -60,26 +60,26 @@ describe('Dynamic form control group component', function () {
         });
     };
 
-    it("renders label if it exists", () => {
+    it("renders psuedo label if it exists", () => {
         const rendered = shallowMount(DynamicFormControlGroup, {
             propsData: {
                 controlGroup: fakeFormGroup
             }
         });
 
-        const labelCol = rendered.find("label");
+        const labelCol = rendered.find(".col-form-label");
         expect(labelCol.text()).toBe("Test 1");
         expect(labelCol.classes()).toStrictEqual(["col-form-label", "col-md-5"]);
     });
 
-    it("does not render label col if there is no label", () => {
```

Comment:
This test did not make much sense to begin with as it is only a shallow mount and there are no columns within this file.